src/python/pants/engine/streaming_workunit_handler.py

```diff
@@ -23,6 +23,8 @@
 
 logger = logging.getLogger(__name__)
 
+ExpandedSpecs = Dict[str, List[str]]
```

Comment:
@gshuflin 
as discussed, maybe make the value a list of dict (or even better, a dataclass) so we can add more data there in the future (beside name), like tags, type, etc...
it is fine if this dataclass only contains a single string attribute for now.
But let us make is expandable w/o breaking the interface.